### ISSUE TITLE
Question related to Fine-Tuning Phase 

### ISSUE THREAD

**Issue by amalislam675** [nadavbra/protein_bert#53]
Hello,
Thanks for this great publication and open source this whole code.
  In the supplementary material of the paper, this thing is mentioned: "for tasks involving global predictions (at the entire protein level), the final layer was connected to its global representations. Specifically, we used a concatenation of all the global normalization layers (2 per block) and the output annotations". I have gone through your notebook that is related to finetuning step. but did not find out the answer What does this above statement mean " It means, will we load the pretrained model weights for the annotation part only and will provide protein sequence and GO annotation (all zero vector) as input  and pass it through a dropout layer with 0.5% probability and then and then pass to dense layer and map it to 8943 dimensions and predict the GO annotations from the protein sequence". Can you explain this fine tuning process? I want to predict GO annotations from protein sequences.

**Comment by nadavbra** [comment]
@amalislam675 The concatenation of all hidden layers is implemented in the code by the `get_model_with_hidden_layers_as_outputs` function (see our [demo notebook](https://github.com/nadavbra/protein_bert/blob/master/ProteinBERT%20demo.ipynb)).

If you're interested in predicting GO annotations, please notice [this issue](https://github.com/nadavbra/protein_bert/issues/6).

**Comment by amalislam675** [comment]
Thanks for your answer. 
Can we predict multiple labels of GO annotations by giving the protein sequence alone to the model?

For example, total GO terms labels are 5 (GO-term1, GO-term2, GO-term3, GO-term4, GO-term5); protein sequence1 is annotated with GO-term1, GO-term2, GO-term3, while protein sequence 2 is annotated with GO-term 2, and GO-term5.

My aim is to give model the protein sequence1, and protein sequnce2 and ask the model to predict the GO-terms with which both protein sequence 1 and protein sequence 2  are annotated, model should predict the labels for protein sequence 1 are [1 1 1 0 0] and for the protein sequence 2  it should predict [0 1 0 0 1]. 
Can we predict multilabel outputs from the proteinBERT by giving them protein sequence alone as input and as label [0 0 0 0 0] all zero annotation vector?

**Comment by amalislam675** [comment]
> @amalislam675 The concatenation of all hidden layers is implemented in the code by the `get_model_with_hidden_layers_as_outputs` function (see our [demo notebook](https://github.com/nadavbra/protein_bert/blob/master/ProteinBERT%20demo.ipynb)).
> 
> If you're interested in predicting GO annotations, please notice [this issue](https://github.com/nadavbra/protein_bert/issues/6).
Thanks for your answer.
I have some other questions.
Can we predict multiple labels of GO annotations by giving the protein sequence alone to the model?

For example, total GO terms labels are 5 (GO-term1, GO-term2, GO-term3, GO-term4, GO-term5); protein sequence1 is annotated with GO-term1, GO-term2, GO-term3, while protein sequence 2 is annotated with GO-term 2, and GO-term5.

My aim is to give model the protein sequence1, and protein sequnce2 and ask the model to predict the GO-terms with which both protein sequence 1 and protein sequence 2 are annotated, model should predict the labels for protein sequence 1 are [1 1 1 0 0] and for the protein sequence 2 it should predict [0 1 0 0 1].
Can we predict multilabel outputs from the proteinBERT by giving them protein sequence alone as input and as label [0 0 0 0 0] all zero annotation vector?
